### PR TITLE
Task/vs extensions

### PR DESCRIPTION
**Pull request by barthamark**

### CODE REVIEW

**Comment by agriffard** [comment]
Tell me if it can be merged it, add :shipit: in a comment

File: docs/Documentation/Visual-Studio-extensions-and-snippets.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Visual Studio Extensions and Snippets
+
+There are already a few Visual Studio extensions and snippets ready to help your everyday Orchard development work.
+
+### Visual Studio Extensions
+
+Orchard-related extensions are listed here in alphabetical order:
+
+  * **Lombiq Orchard Visual Studio Extension**: Visual Studio extension with many features and templates frequently used by the [Lombiq](http://lombiq.com) developers. Contains Orchard-related as well as generic goodies. Repository: [https://github.com/Lombiq/Lombiq-Orchard-Visual-Studio-Extension](https://github.com/Lombiq/Lombiq-Orchard-Visual-Studio-Extension).
+  * **Orchardizer**: an extension for Visual Studio 2013 by [Harry Westbrook](http://arkleseizure.net) that reduces development of Orchard modules to a bunch of textboxes and some sporadic clicks. Repository: [https://github.com/Hazzamanic/orchardizer](https://github.com/Hazzamanic/orchardizer).
```
Comment (rtpHarry) [inline docs/Documentation/Visual-Studio-extensions-and-snippets.markdown]: This should be Visual Studio 2013 and above. The repo doesn't state it but I'm using it with Visual Studio Community 2015.

File: docs/Documentation/Visual-Studio-extensions-and-snippets.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  * **VS Constructor Field Refactoring rule**: an extension by [Sebastien Ros](http://sebastienros.com/) to generate a private readonly field based on a constructor parameter. Repository: [https://github.com/sebastienros/vsconstructorfield](https://github.com/sebastienros/vsconstructorfield).
```
Comment (rtpHarry) [inline docs/Documentation/Visual-Studio-extensions-and-snippets.markdown]: Small point, his name is Sébastien with the squiggle. I think the anglicised version is acceptable but looking at his social profiles he does use it consistently that way.

File: docs/Documentation/Visual-Studio-extensions-and-snippets.markdown (hunk 1 above)
Comment (barthamark) [inline docs/Documentation/Visual-Studio-extensions-and-snippets.markdown]: Thanks, I fix it.

File: docs/Documentation/Visual-Studio-extensions-and-snippets.markdown (hunk 2 above)
Comment (barthamark) [inline docs/Documentation/Visual-Studio-extensions-and-snippets.markdown]: You're right. Fixed.

**Comment by Piedone** [comment]
:shipit: by me

**Comment by agriffard** [comment]
@Piedone : I think you (@LombiqTechnologies) could be added in the OrchardDoc Team to make contributions easier.

**Comment by Piedone** [comment]
Previously we discussed this (on the Orchard meeting) and the conclusion was to only give specific people access (but we'll continue to use @LombiqTechnologies as the author), so it's OK like this. And PRs are useful anyway.

**Comment by rtpHarry** [comment]
Yeah this was a productive review

**Pull request merged**